### ISSUE TITLE
Tinymce not working with ManifestStaticFilesStorage

### ISSUE THREAD

**Issue by nikolaysm** [romanvm/django-tinymce4-lite#63]
Hi everyone, I am using django 1.11, django-tinymce4-lite 1.7.4. 

**### This problem only occurs in internet explorer 11.**

`ManifestStaticFilesStorage` is a subclass of the `StaticFilesStorage` storage backend which stores the file names it handles by appending the MD5 hash of the file’s content to the filename. For example, the file `js/tinymce.min.js` would also be saved as `js/tinymce.min.500c07cb726a.js`.

If I switch my storage to `StaticFilesStorage` it works fine.

Has anyone come across that yet? Or a suggestion how that can be solved (with `ManifestStaticFilesStorage` to keep)

**Comment by romanvm** [comment]
"Not working" is a rather vague description. Please describe your issue with as much details as possible.

**Comment by nikolaysm** [comment]
So no error in console or anywhere else. Textarea is not displayed, no tinymce wrapper added.

**Comment by merwok** [comment]
Is this the problem: The generated HTML contains links like `js/tinymce.min.js` but it should be `ss/tinymce.min.500c07cb726a.js` to match how the storage class saved it

(Probably not, as you wrote that the problem is browser-specific, which wouldn’t be the case with that)

**Comment by nikolaysm** [comment]
Yes, the problem only occurs in internet explorer and only when I use renamed tinymce.js.
Actually the problem has nothing to do with `ManifestStaticFilesStorage`.
